addons/ammunition/CfgAmmo.hpp

```diff
@@ -1,5 +1,7 @@
 class CfgAmmo {
     class BulletBase;
+    class B_556x45_Ball;
+    class B_762x39_Ball_F;
     class B_12Gauge_Pellets_Submunition: BulletBase {};
```

Comment:
```suggestion
    class B_12Gauge_Pellets_Submunition;
```
Doesn't this also work?